docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -56,6 +56,33 @@ Different settings can be applied to a load balancer service in AWS using _annot
 
 The information for the annotations for AWS is taken from the comments on [aws.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/aws.go)
 
+# OPENSTACK
+This section describes all the possible configurations which can
+be used while setting Kubernetes as an OpenStack provider.
+
+## cloud.conf
+Kubernetes configuration is modified via the file cloud.conf.It is the file that will tell Kubernetes where are your credentials and OpenStack auth endpoint.
```

Comment:
Kubernetes knows how to interact with OpenStack via the cloud.conf file. It is this file that ...